web/impact/impact/graphql/types/expert_profile_type.py

```diff
@@ -77,6 +85,28 @@ def resolve_office_hours_url(self, info, **kwargs):
                 '?mentor_id={mentor_id}'.format(
                     mentor_id=self.user.id))
 
+    def resolve_current_mentees(self, info, **kwargs):
+        mentee_filter = compose_filter([
+            'startup_mentor_tracking',
+            'program',
+            'program_status'
+        ], ACTIVE_PROGRAM_STATUS)
+        return self.user.startup_mentor_relationships.filter(
+            status=CONFIRMED_RELATIONSHIP,
+            **mentee_filter
+            )
+
+    def resolve_previous_mentees(self, info, **kwargs):
```

Comment:
This function and the previous one differ only in the PROGRAM_STATUS that they select on. I would like to see them combined into a single function, taking the PROGRAM_STATUS as an argument.